File: Assignment6/program4.c
////////////////////////////////////////////////////////////////
//
//  File Name   : program4.c
//  Description : Accept a Integer and returns it Multiplication Table
//  Author      : Akhilesh.P.Sonavane.
//  Date        : 17/05/2025
//
////////////////////////////////////////////////////////////////

#include<stdio.h>

////////////////////////////////////////////////////////////////
// 
//  Function Name : Table
//  Description   : Returns Multiplication Table of input Integer
//  Input         : Integer
//  Output        : Integer
//
////////////////////////////////////////////////////////////////

void Table(int iNo)
{
    int iNum = 1;
   
    if(iNo < 0)
   {
    iNo = -iNo;
   }
   
   for (int iCnt = 1; iCnt <= 10; iCnt++)
   {
       iNum=iNo * iCnt;
       printf("%d\n",iNum);
   }

}

int main()
{
    int iValue = 0;
    
    printf("Enter number");
    scanf("%d",&iValue);
    
    Table(iValue);

    return 0;
}

File: Assignment19/program4.java
////////////////////////////////////////////////////////////////
//
//  File Name   : program4.java
//  Description : Accept a N Integer and returns Frequency of 11 in all elements 
//  Author      : Akhilesh.P.Sonavane.
//  Date        : 25/05/2025
//
////////////////////////////////////////////////////////////////

import java.util.Scanner;

////////////////////////////////////////////////////////////////
// 
//  Class Name    : Display
//  Function Name : CheckFrequency
//  Description   : Inputs N Integer and returns Frequency of 11 in all elements
//  Input         : Integer
//  Output        : Integer
//
////////////////////////////////////////////////////////////////

class Display
{
    public int CheckFrequency(int[] iPtr, int iSize)
    {
        int iCnt = 0;
        int iFreq = 0;
     
        for(iCnt = 0; iCnt < iSize; iCnt++)
        {
            if(iPtr[iCnt] == 11)
            {
               iFreq = (iCnt+1);
               break;
            }
        }
        return iFreq;
    }
}
public class program4
{
    public static void main(String args[])
    {
        int[] iArr;
        int iLength = 0;
        int iCnt = 0;
        int iRet = 0;
    
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter Number of Elements : ");
        iLength = sobj.nextInt();

        iArr = new int[iLength];

        for(iCnt = 0; iCnt < iLength; iCnt++)
        {
            System.out.print("Element "+(iCnt+1)+" : ");
            iArr[iCnt] = sobj.nextInt();
        }

        Display dobj = new Display();

        iRet = dobj.CheckFrequency(iArr, iLength);

        System.out.println("Frequency of 11 from "+iLength+" elements is :  "+iRet);
        
        sobj.close();
    }
} 